extide/gradle/netbeans-gradle-tooling/src/main/java/org/netbeans/modules/gradle/api/NbProjectInfo.java

```diff
@@ -41,5 +41,22 @@
      */    
     Map<String, Object> getExt();
     Set<String> getProblems();
+
+    /**
+     * @since 2.23
+     */
+    Set<Report> getReports();
+    
     boolean getMiscOnly();
+    
+    /**
+     * @since 2.23
+     */
+    interface Report {
+        public String getErrorClass();
+        public String getScriptLocation();
```

Comment:
It would be useful to have a JavaDoc here, without an explanation seems to be strange that a method called **getScriptLocation()** returns a String.

Reply:
According to @lkishalmi assessment, this should not be treated as an API, but rather removed :)